entity_sync.module

```diff
@@ -75,10 +83,24 @@ function entity_sync_export_local_entity_all_syncs(
       continue;
     }
 
-    // @I Support queuing local entity exports
-    //    type     : improvement
-    //    priority : normal
-    //    labels   : export, performance
-    $manager->exportLocalEntity($sync->get('id'), $entity);
+    // If the config wants to queue the export, do so. Otherwise, perform the
+    // export immediately.
+    $sync_queue_key = 'operations.export_entity.update_entities.queue';
+    if ($op === 'insert') {
+      $sync_queue_key = 'operations.export_entity.create_entities.queue';
+    }
```

Comment:
**CHANGE REQUEST**

We can use the operation for easily generating the key: `$sync_queue_key = "operations.export_entity.{$op}_entities.queue";`

However, remember the discussion we had about updating on-the-fly for inserts being tricky and that we will leave handling that for later. That means that we don't support that for now and the config property value has to be `FALSE`. If not, throw an `InvalidConfigurationException`.